src/Esfa.Vacancy.Register.UnitTests/Api/Orchestrators/VacancyOrchestratorTests/WhenGettingLiveVacancyWithApprenticeshipMinimumWageType.cs

```diff
@@ -0,0 +1,96 @@
+        {
+            const decimal minApprenticeshipWageRate = 4.05m;
+            const decimal hoursPerWeek = 30;
+
+            _mockMediator.Setup(m => m.Send(It.IsAny<GetVacancyRequest>(), CancellationToken.None))
+                .ReturnsAsync(new GetVacancyResponse
+                {
+                    Vacancy = new Fixture().Build<Domain.Entities.Vacancy>()
+                                            .With(v => v.VacancyReferenceNumber, VacancyReference)
+                                            .With(v => v.VacancyStatusId, LiveVacancyStatusId)
+                                            .With(v => v.VacancyTypeId, (int)VacancyType.Apprenticeship)
+                                            .With(v => v.WageType, (int)WageType.ApprenticeshipMinimum)
+                                            .With(v => v.MinimumWageRate, minApprenticeshipWageRate)
+                                            .With(v => v.HoursPerWeek, hoursPerWeek)
+                                            .Without(v => v.WageUnitId)
+                                            .Create()
+                });
+
+            var result = await _sut.GetVacancyDetailsAsync(VacancyReference);
+
+            result.VacancyReference.Should().Be(VacancyReference);
+            result.VacancyType.Should().Be(VacancyType.Apprenticeship);
+            result.WageUnit.Should().BeNull();
+            result.Wage.Should().Be("£121.50");
+        }
+
+        [Test]
+        public async Task ShouldHaveUnknownWageForVacancyWithUndefinedHoursPerWeek()
+        {
+            const decimal minApprenticeshipWageRate = 4.05m;
+
+            _mockMediator.Setup(m => m.Send(It.IsAny<GetVacancyRequest>(), CancellationToken.None))
+                .ReturnsAsync(new GetVacancyResponse
+                {
+                    Vacancy = new Fixture().Build<Domain.Entities.Vacancy>()
+                                            .With(v => v.VacancyReferenceNumber, VacancyReference)
+                                            .With(v => v.VacancyStatusId, LiveVacancyStatusId)
+                                            .With(v => v.VacancyTypeId, (int)VacancyType.Apprenticeship)
+                                            .With(v => v.WageType, (int)WageType.ApprenticeshipMinimum)
+                                            .With(v => v.MinimumWageRate, minApprenticeshipWageRate)
+                                            .Without(v => v.HoursPerWeek)
+                                            .Without(v => v.WageUnitId)
+                                            .Create()
+                });
+
+            var result = await _sut.GetVacancyDetailsAsync(VacancyReference);
+
+            result.VacancyReference.Should().Be(VacancyReference);
+            result.VacancyType.Should().Be(VacancyType.Apprenticeship);
+            result.WageUnit.Should().BeNull();
+            result.Wage.Should().Be(UnknownwWageText);
+        }
+    }
```

Comment:
Do we need a Test here for if the minApprenticeshipWageRate is null but hours are defined?

Reply:
This is from a reference table and the minApprenticeshipWageRate would not be null.